ioxd_router2/src/lib.rs

```diff
@@ -174,6 +176,10 @@ pub async fn create_router2_server_type(
         &*metrics,
     ));
 
+    pre_warm_schema_cache(&ns_cache, &*catalog)
```

Comment:
Is it worth a note here about why prewarming the cache is needed? Like

```suggestion
    // load schemas / tables / columns into the table prior to serving
    // to avoid dropping writes
    // https://github.com/influxdata/influxdb_iox/issues/3577
    pre_warm_schema_cache(&ns_cache, &*catalog)
```